perses/tests/test_smc.py

```diff
@@ -0,0 +1,143 @@
+###########################################
+# IMPORTS
+###########################################
+from simtk.openmm import app
+from simtk import unit, openmm
+import numpy as np
+import os
+from nose.tools import nottest
+from unittest import skipIf
+from perses.app.setup_relative_calculation import *
+from perses.annihilation.relative import HybridTopologyFactory
+from perses.dispersed import parallel
+from perses.rjmc.topology_proposal import SmallMoleculeSetProposalEngine
+from perses.app.relative_setup import RelativeFEPSetup
+from perses.tests.utils import validate_endstate_energies
+from perses.dispersed.smc import SequentialMonteCarlo
+from simtk import openmm, unit
+from openmmtools.constants import kB
+#######################
+temperature = 300.0 * unit.kelvin
```

Comment:
Does this need to be hard-coded here?

Reply:
It's also repeated on line 49

Reply:
fixed; it is hard coded because they are the params used for all of the full AIS/sMC tests